### PR TITLE
content update

### PR DESCRIPTION
**Pull request by DannyS03**
Addresses: https://github.com/OpenLiberty/guide-rest-intro/issues/20

### CODE REVIEW

**Comment by gkwan-ibm** [comment]
hi @andymc12 Would you help to review this PR? If need, you can visit `http://lgdev1.fyre.ibm.com:4060/guides/rest-intro.html`

File: README.adoc [hunk 1]
```diff
@@ -49,13 +49,11 @@ each property is a field in a JSON object like this:
...
 ----
 
-When you create a new REST application, the design of the API is important. The JAX-RS APIs can be
-used to create JSON-RPC, or XML-RPC APIs, but it wouldn't be a RESTful service. A good RESTful service
-is designed around the resources that are exposed, and on how to create, read, update, and delete the
-resources.
-
-The service responds to `GET` requests to the `/System/properties` path. The `GET` request should
-return a `200 OK` response that contains all of the JVM's system properties.
+The design of an HTTP API is essential when creating a web application. The REST API has exclusively
```
Comment (andymc12) [inline README.adoc:52]: I would drop the word "exclusively".  Newer HTTP-based technologies, like GraphQL and gRPC are gaining in popularity.

File: README.adoc [hunk 2]
```diff
@@ -138,9 +135,7 @@ For this application, a JSON structure is to be returned. The desired `Content-T
 response is `application/json` with `MediaType.APPLICATION_JSON` instead of the `String` content type. Using a constant such as `MediaType.APPLICATION_JSON` is better because if there's a spelling error, a compile failure occurs.
 
 JAX-RS supports a number of ways to marshal JSON. The JAX-RS 2.1 specification mandates JSON-Binding
-(JSON-B) and JAX-B. 
-
-The method body returns the result of `System.getProperties()` that is of type `java.util.Properties`. Since the method 
+(JSON-B) and JAX-B. The method body returns the result of `System.getProperties()` that is of type `java.util.Properties`. Since the method
```
Comment (andymc12) [inline README.adoc:138]: The spec does mandate JAXB, but JAXB is intended for Java-to-XML (and vice versa) transformations, not Java-to-JSON.  I would suggest removing the reference to it here.

File: README.adoc [hunk 3]
```diff
@@ -209,11 +204,7 @@ http://localhost:9080/LibertyProject/System/properties[^] URL.
 
 == Testing the service
 
-You can test this service manually by starting a server and pointing a web browser at the
-http://localhost:9080/LibertyProject/System/properties[^] URL. Automated tests are a much better
-approach because they trigger a failure if a change introduces a bug. JUnit and the JAX-RS Client
-API provide a simple environment to test the application.
-
+JUnit and the JAX-RS Client API provide a simple environment to test the application.
```
Comment (andymc12) [inline README.adoc:207]: Is there a reason to remove the text about testing it in a browser?  I agree with the text that automated tests like JUnit are superior, but it is often nice to be able to manually test out services - and the browser is a great way to do it.

Comment (DannyS03) [reply README.adoc]: Addressed.

File: README.adoc (hunk 2 above)
Comment (DannyS03) [reply README.adoc]: Addressed.

File: README.adoc (hunk 1 above)
Comment (DannyS03) [reply README.adoc]: Addressed.

**Comment by gkwan-ibm** [comment]
hi @Charlotte-Holt would you review this PR?

File: README.adoc [hunk 4]
```diff
@@ -49,13 +49,11 @@ each property is a field in a JSON object like this:
...
-is designed around the resources that are exposed, and on how to create, read, update, and delete the
-resources.
-
-The service responds to `GET` requests to the `/System/properties` path. The `GET` request should
-return a `200 OK` response that contains all of the JVM's system properties.
+The design of an HTTP API is essential when creating a web application. The REST API has 
+become the go-to architectural style for building an HTTP API. The JAX-RS API offers functionality 
+on creating, reading, updating, and deleting exposed resources. The JAX-RS API supports the creation 
+of RESTful web services that induce desirable properties, such as performance, scalability, and 
+modifiability.
```
Comment (Charlotte-Holt) [inline README.adoc:56]: Update

>The JAX-RS API offers functionality on creating, reading, updating, and deleting exposed resources. The JAX-RS API supports the creation of RESTful web services that induce desirable properties, such as performance, scalability, and modifiability.

To

>The JAX-RS API offers functionality for creating, reading, updating, and deleting exposed resources. The JAX-RS API supports the creation of RESTful web services that come with desirable properties, such as performance, scalability, and modifiability.

File: README.adoc [hunk 5]
```diff
@@ -91,10 +89,9 @@ common path. You can think of the JAX-RS application as a wrapper for all of you
 `src/main/java/io/openliberty/guides/rest/SystemApplication.java`
 ----
 
-The [hotspot=systemApplication]`SystemApplication` class extends the `Application` class, which in turn associates all JAX-RS
-resource classes in the WAR file with this JAX-RS application, making them available under the common
-path specified in the [hotspot=systemApplication]`SystemApplication` class. The [hotspot=applicationPath]`@ApplicationPath` annotation has a
-value that indicates the path within the WAR that the JAX-RS application accepts requests from.
+The [hotspot=systemApplication]`SystemApplication` class extends the `Application` class, which 
+in turn associates all JAX-RS resource classes in the WAR file with this JAX-RS application. These resources become available under the common path specified using the [hotspot=applicationPath]`@ApplicationPath` 
+annotation. The `@ApplicationPath` annotation has a value that indicates the path within the WAR that the JAX-RS application accepts requests from.
```
Comment (Charlotte-Holt) [inline README.adoc:94]: Update

>The `SystemApplication` class extends the `Application` class, which in turn associates all JAX-RS resource classes in the WAR file with this JAX-RS application. These resources become available under the common path specified using the `@ApplicationPath` annotation. The `@ApplicationPath` annotation has a value that indicates the path within the WAR that the JAX-RS application accepts requests from.

To

>The `SystemApplication` class extends the `Application` class, which associates all JAX-RS resource classes in the WAR file with this JAX-RS application. These resources become available under the common path that's specified with the `@ApplicationPath` annotation. The `@ApplicationPath` annotation has a value that indicates the path in the WAR file that the JAX-RS application accepts requests from.

File: README.adoc [hunk 6]
```diff
@@ -127,9 +124,9 @@ in the JAX-RS application. The [hotspot=applicationPath file=1]`@ApplicationPath
 the [hotspot=path file=0]`@Path` annotation in this class indicates that the resource is available at the `System/properties`
 path.
 
-JAX-RS maps the HTTP methods on the URL to the methods on the class. The method to call is determined
-by the annotations that are specified on the methods. In the application you are building, an HTTP `GET` request
-to the `System/properties` path results in the system properties being returned.
+JAX-RS maps the HTTP methods on the URL to the methods of the class using annotations.
```
Comment (Charlotte-Holt) [inline README.adoc:127]: Update

>JAX-RS maps the HTTP methods on the URL to the methods of the class using annotations. 

To

>JAX-RS maps the HTTP methods on the URL to the methods of the class by using annotations.

File: README.adoc [hunk 7]
```diff
@@ -138,9 +135,7 @@ For this application, a JSON structure is to be returned. The desired `Content-T
 response is `application/json` with `MediaType.APPLICATION_JSON` instead of the `String` content type. Using a constant such as `MediaType.APPLICATION_JSON` is better because if there's a spelling error, a compile failure occurs.
 
 JAX-RS supports a number of ways to marshal JSON. The JAX-RS 2.1 specification mandates JSON-Binding
-(JSON-B) and JAX-B. 
-
-The method body returns the result of `System.getProperties()` that is of type `java.util.Properties`. Since the method 
+(JSON-B). The method body returns the result of `System.getProperties()` that is of type `java.util.Properties`. Since the method
```
Comment (Charlotte-Holt) [inline README.adoc:138]: Update

>The method body returns the result of `System.getProperties()` that is of type `java.util.Properties`.

To

>The method body returns the result of `System.getProperties()`, which is of type `java.util.Properties`.

File: README.adoc [hunk 8]
```diff
@@ -287,6 +282,6 @@ include::{common-includes}/devmode-quit.adoc[]
 
 == Great work! You're done!
 
-You developed a REST service in Open Liberty by using JAX-RS and JSON-B.
+You have just developed a REST service in Open Liberty by using JAX-RS and JSON-B.
```
Comment (Charlotte-Holt) [inline README.adoc:285]: Update

>You have just developed a REST service in Open Liberty by using JAX-RS and JSON-B.

To

>You just developed a REST service in Open Liberty by using JAX-RS and JSON-B.

File: README.adoc (hunk 4 above)
Comment (DannyS03) [reply README.adoc]: Addressed.

File: README.adoc (hunk 5 above)
Comment (DannyS03) [reply README.adoc]: Addressed.

File: README.adoc (hunk 6 above)
Comment (DannyS03) [reply README.adoc]: Addressed.

File: README.adoc (hunk 7 above)
Comment (DannyS03) [reply README.adoc]: Addressed.

File: README.adoc (hunk 8 above)
Comment (DannyS03) [reply README.adoc]: Addressed.

**Comment by DannyS03** [comment]
> @DannyS03 I added ID review feedback. Let me know if you have any questions. Thanks!

Thanks, @Charlotte-Holt. You can verify the changes :)

File: README.adoc [hunk 9]
```diff
@@ -138,9 +135,7 @@ For this application, a JSON structure is to be returned. The desired `Content-T
 response is `application/json` with `MediaType.APPLICATION_JSON` instead of the `String` content type. Using a constant such as `MediaType.APPLICATION_JSON` is better because if there's a spelling error, a compile failure occurs.
 
 JAX-RS supports a number of ways to marshal JSON. The JAX-RS 2.1 specification mandates JSON-Binding
-(JSON-B) and JAX-B. 
-
-The method body returns the result of `System.getProperties()` that is of type `java.util.Properties`. Since the method 
+(JSON-B). The method body returns the result of `System.getProperties()` which is of type `java.util.Properties`. Since the method
```
Comment (Charlotte-Holt) [inline README.adoc:138]: Add a comma after `System.getProperties()` in "The method body returns the result of `System.getProperties()` which is of type `java.util.Properties`."

Comment (DannyS03) [reply README.adoc]: Addressed.

**Comment by AustinSeto** [comment]
Checked README changes on QA site, all looks good to me

**Pull request merged**